.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: You have an idea on how Tatoeba could be improved
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+*Please tell us a story.* How did you came up with that idea? What made you think about it?
```

Comment:
We could turn this into a section:

> **Story** or **Context**
> Tell us a story. How did you come up with that idea? What made you think about it?
>
> *Example:*
> ...

then add an additional one:

> **Idea**
> Describe your idea. If possible, include images so that it's easier for us to picture it.